exercises/saddle-points/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+
+pub fn find_saddle_points(input: Vec<Vec<u64>>) -> Vec<(u64, u64)>{
```

Comment:
Naming the variable `input` causes a compiler warning due to the unused variable. While this doesn't break CI, I'd prefer to skip that by renaming the variable to `_input` or simply `_`.